tree/dataframe/test/dataframe_simple.cxx

```diff
@@ -726,12 +726,21 @@ TEST_P(RDFSimpleTests, StandardDeviationEmpty)
    auto stdDev = rd1.Define("b1", []() { return 0; }).StdDev("b1");
    EXPECT_DOUBLE_EQ(*stdDev, 0);
 }
-
+/*
 TEST(RDFSimpleTests, SumOfStrings)
 {
    auto df = RDataFrame(2).Define("str", []() -> std::string { return "bla"; });
    EXPECT_EQ(*df.Sum<std::string>("str"), "blabla");
 }
+*/
+
+
+TEST(RDFSimpleTests, KahanSum)
+{
+   ROOT::RDataFrame d(20);
+   auto dd = d.Define("x", "(rdfentry_ %2 == 0) ? 0.00000001 : 100000000.");
+   EXPECT_EQ(*dd.Sum<double>({"x"}), 1000000000.00000011920929);
```

Comment:
as discussed I am not a big fan of this particular test setup as the errors are at the 17th significant digit.

it would be better to have a test for `float`s and a test for `double`s that show numerical instability at digits well within their expected precision.

also testing `double`s for equality is very tricky (on different architectures we might get slightly different results). it would be better to use `EXPECT_DOUBLE_EQ` or `EXPECT_NEAR`, see also https://google.github.io/googletest/reference/assertions.html#floating-point